devservices/lambda/src/main/java/io/quarkus/amazon/devservices/lambda/LambdaDevServicesProcessor.java

```diff
@@ -0,0 +1,84 @@
+package io.quarkus.amazon.devservices.lambda;
+
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+
+import org.testcontainers.containers.localstack.LocalStackContainer;
+import org.testcontainers.containers.localstack.LocalStackContainer.Service;
+
+import io.quarkus.amazon.common.deployment.spi.AbstractDevServicesLocalStackProcessor;
+import io.quarkus.amazon.common.deployment.spi.DevServicesLocalStackProviderBuildItem;
+import io.quarkus.amazon.common.deployment.spi.LocalStackDevServicesBaseConfig;
+import io.quarkus.amazon.common.runtime.DevServicesBuildTimeConfig;
+import io.quarkus.amazon.lambda.client.runtime.LambdaBuildTimeConfig;
+import io.quarkus.amazon.lambda.client.runtime.LambdaDevServicesBuildTimeConfig;
+import io.quarkus.deployment.annotations.BuildStep;
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.core.SdkBytes;
+import software.amazon.awssdk.http.urlconnection.UrlConnectionHttpClient;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.lambda.LambdaClient;
+import software.amazon.awssdk.services.lambda.model.Runtime;
+
+public class LambdaDevServicesProcessor extends AbstractDevServicesLocalStackProcessor {
+
+    public static final String HELLO_LAMBDA = "hello-lambda";
+
+    @BuildStep
+    @SuppressWarnings("unused")
+    DevServicesLocalStackProviderBuildItem setupLambda(LambdaBuildTimeConfig clientBuildTimeConfig) {
+        return this.setup(Service.LAMBDA, clientBuildTimeConfig.devservices());
+    }
+
+    @Override
+    protected void prepareLocalStack(DevServicesBuildTimeConfig clientBuildTimeConfig, LocalStackContainer localstack) {
+        createFunctions(localstack, getConfiguration((LambdaDevServicesBuildTimeConfig) clientBuildTimeConfig));
+    }
+
+    @SuppressWarnings("unused")
+    public void createFunctions(LocalStackContainer localstack, LambdaDevServiceCfg configuration) {
+        try (LambdaClient client = LambdaClient.builder()
+                .endpointOverride(localstack.getEndpointOverride(Service.LAMBDA))
+                .region(Region.of(localstack.getRegion()))
+                .credentialsProvider(StaticCredentialsProvider.create(AwsBasicCredentials
+                        .create(localstack.getAccessKey(), localstack.getSecretKey())))
+                .httpClientBuilder(UrlConnectionHttpClient.builder())
+                .build()) {
+            client.createFunction(b -> b.functionName(HELLO_LAMBDA)
```

Comment:
Should be moved to either `test` or `integration-tests` (preferably the latter in this case, as the class will be much empty, similar to SSM).

Reply:
Addressed

Reply:
Moved to IT